pkg/workload/schemachange/schemachange.go

```diff
@@ -585,26 +604,27 @@ func (w *schemaChangeWorker) run(ctx context.Context) error {
 					errors.Wrapf(err, "***UNEXPECTED COMMIT ERROR; Received an unexpected commit error")),
 				errRunInTxnFatalSentinel,
 			)
-			w.logger.flushLogWithError(tx, err)
+			w.logger.flushLogWithError(err)
 			w.preErrorHook()
 			return err
 		}
 
 		// Error was anticipated, so it is acceptable.
 		w.recordInHist(timeutil.Since(start), txnCommitError)
-		w.logger.flushLog(tx, "COMMIT; Successfully got expected commit error")
+		w.logger.flushLog("COMMIT; Successfully got expected commit error")
 		return nil
 	}
 	if !w.opGen.expectedCommitErrors.empty() {
 		err := w.WrapWithErrorState(errors.Newf("***FAIL; Failed to receive a commit error when at least one commit error was expected"))
-		w.logger.flushLogWithError(tx, err)
+		w.logger.flushLogWithError(err)
 		w.preErrorHook()
 		return errors.Mark(err, errRunInTxnFatalSentinel)
 	}
 
 	// If there were no errors while committing the txn.
-	w.logger.flushLog(tx, "")
+	w.logger.flushLog("")
 	w.recordInHist(timeutil.Since(start), txnOk)
+	workloadMetrics[txnCommitted] = attribute.BoolValue(true)
```

Comment:
Is this `workloadMetrics` capturing all transactions ever run within one worload worker? If so, why is "txnCommitted" a boolean valued attribute that is either `True` or `False`?

I though a worker could perform many transactions, some succeeded and some failed, so we ought to store the number of succeeded/failed txns ever performed within this work.

But I felt like I missed something here....